src/prepack-options.js

```diff
@@ -103,10 +104,9 @@ export function getSerializerOptions({
   return result;
 }
 
-export function getDebuggerOptions({
-  debugInFilePath = "./src/debugger/.sessionlogs/adapter2engine.txt",
-  debugOutFilePath = "./src/debugger/.sessionlogs/engine2adapter.txt",
-}: PrepackOptions): DebuggerOptions {
+export function getDebuggerOptions({ debugInFilePath, debugOutFilePath }: PrepackOptions): DebuggerOptions {
+  // if the debugger is started, the in and out files need to be specified
```

Comment:
I don't understand this comment in connection with the invariant below which seems to enforce that they do already have to be specified.

Reply:
This comment just explains why this invariant is here.

Reply:
Style-wise prefer breaking && checks into multiple invariants (clearer to see which one failed when debugging) and putting comments as invariant messages (2nd arg) if necessary. In this case the comment seems redundant with the invariant.

Reply:
Updated invariants and removed comment.